recipes/zoxide/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "zoxide" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ajeetdsouza/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 62b7a2ced73d5ac0a183b3855d54d6619166b4d8d8c74299bb610265ccc4b193
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+
+test:
+  commands:
+    - zoxide --help
+
+about:
+  home: https://github.com/ajeetdsouza/zoxide
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
The licenses of the dependencies need to be included for rust packages. See https://github.com/conda-forge/conda-forge.github.io/issues/1052 for a starting point

Reply:
trying to follow [this](https://github.com/conda-forge/staged-recipes/pull/13208#issuecomment-726767185)

Reply:
@chrisburr done with the license files. Please take a look, thanks!